.ansible-pulp_tox.sh

```diff
@@ -9,10 +9,10 @@ if [ ! -e roles/pulp.pulp_rpm_prerequisites ]; then
   ln -s $TRAVIS_BUILD_DIR roles/pulp.pulp_rpm_prerequisites
 fi
 
-find */group_vars/all -exec sh -c "yq w -i {} pulp_use_system_wide_pkgs true" \;
-find */group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.app_label rpm" \;
-find */group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.source_dir git+https:\/\/github.com\/pulp\/pulp_rpm.git#egg=pulp-rpm" \;
-find */group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.prereq_role pulp.pulp_rpm_prerequisites" \;
+find ./molecule/*/group_vars/all -exec sh -c "yq w -i {} pulp_use_system_wide_pkgs true" \;
+find ./molecule/*/group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.app_label rpm" \;
+find ./molecule/*/group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.source_dir git+https:\/\/github.com\/pulp\/pulp_rpm.git#egg=pulp-rpm" \;
+find ./molecule/*/group_vars/all -exec sh -c "yq w -i {} pulp_install_plugins.pulp-rpm.prereq_role pulp.pulp_rpm_prerequisites" \;
```

Comment:
Looks good, just review the output to make sure it works, since group_vars is sometimes a symlink.

Reply:
Let's run this again, and review output, after I merge https://github.com/pulp/ansible-pulp/pull/218/commits/5b31a7321e954d1c05270b3bd6ee7c5709977fac . Since it reworks a lot of these files / symlinks.